src/metrics.js

```diff
@@ -0,0 +1,81 @@
+const octokit = require('@octokit/rest')() 
+const fs = require('fs');
+ 
+var gitToken = "thisToken"
```

Comment:
This needs to come from an environment variable or the like, you can't leave it like this.  Also, you're not using it below, which you should.